handsontable/src/core/viewportScroll/__tests__/multipleSelection.spec.js

```diff
@@ -337,9 +345,12 @@ describe('Multiple selection scroll', () => {
       keyDown('shift');
       simulateClick(getCell(11, 0));
 
-      expect(topOverlay().getScrollPosition()).forThemes(({ classic, main }) => {
+      await sleep(10);
+
+      expect(topOverlay().getScrollPosition()).forThemes(({ classic, main, horizon }) => {
         classic.toBe(19);
         main.toBe(94);
+        horizon.toBe(198);
```

Comment:
this is definitely a constant as `horizon.toBe(198);` appears 13 times in this PR